### PR TITLE
Add needed LDFLAGS for Tensorflow lite backend

### PR DESCRIPTION
**Pull request by lumurillo**
Added missing linkages to libraries needed by the Tensorflow-Lite backend. Issue #69

### CODE REVIEW

File: r2i/tflite/Makefile.am [hunk 1]
```diff
@@ -43,6 +43,9 @@ libtflite_la_LIBADD =                \
     $(TFLITE_LIBS)           \
     $(CODE_COVERAGE_LIBS)
 
+libtflite_la_LDFLAGS =		     \
```
Comment (emmadrigal) [inline r2i/tflite/Makefile.am:46]: Can you make sure that all `\` are in the same column in the file?

File: tests/unit/r2i/tflite/Makefile.am [hunk 2]
```diff
@@ -54,6 +54,7 @@ AM_LDFLAGS =                        \
         $(TFLITE_LIBS)      \
         $(TESTS_LIBS)               \
         $(CODE_COVERAGE_LIBS)       \
+        $(TFLITE_LDFLAGS)           \
```
Comment (emmadrigal) [inline tests/unit/r2i/tflite/Makefile.am:57]: Can you make sure that all \ are in the same column in the file?

File: r2i/tflite/Makefile.am (hunk 1 above)
Comment (rrcarlosrodriguez) [reply r2i/tflite/Makefile.am:46]: Agree

File: configure.ac [hunk 3]
```diff
@@ -52,6 +52,10 @@ RR_CHECK_FEATURE_LIB(TENSORFLOW, TensorFlow Installation,
 AC_LANG_PUSH([C++])
 RR_CHECK_FEATURE_LIB(TFLITE, TensorFlow lite Installation,
                      tensorflow-lite, TfLiteTypeGetName, tensorflow/lite/model.h, no)
+
+# add specific LDFLAGS for TFLITE
```
Comment (michaelgruner) [inline configure.ac:56]: LDFLAGS->LIBS

File: tests/unit/r2i/tflite/Makefile.am [hunk 4]
```diff
@@ -36,22 +36,22 @@ LOG_DRIVER=$(top_builddir)/common/rr-test-driver.py $(SINGLE_PROCESS)
...
 
 AM_CPPFLAGS =                       \
         $(RR_CPPFLAGS)              \
         $(CODE_COVERAGE_CPPFLAGS)
 
-AM_LDFLAGS =                        \
+LDADD =                             \
         $(RR_LIBS)                  \
-        $(TFLITE_LIBS)      \
+        $(TFLITE_LIBS)              \
```
Comment (michaelgruner) [inline tests/unit/r2i/tflite/Makefile.am:54]: This should no longer be necessary. Libtool should handle the inter-library dependencies.

File: examples/r2i/tflite/Makefile.am [hunk 5]
```diff
@@ -27,8 +27,9 @@ AM_CPPFLAGS =                      \
         $(RR_CPPFLAGS)             \
         $(CODE_COVERAGE_CPPFLAGS)
 
-AM_LDFLAGS =                       \
+LDADD =                            \
         $(RR_LIBS)                 \
+        $(TFLITE_LIBS)             \
```
Comment (michaelgruner) [inline examples/r2i/tflite/Makefile.am:32]: This should no longer be necessary. Libtool should handle the inter-library dependencies.

**Pull request closed without merging**